app/controllers/system_admin/pending_awards_controller.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module SystemAdmin
+  class PendingAwardsController < ApplicationController
+    add_flash_types :dqt_error
+
+    helper_method :last_run_or_scheduled_at, :job_status
+
+    def index
+      @trainees = Trainee.recommended_for_award.undiscarded.order(:recommended_for_award_at)
+    end
+
+    def last_run_or_scheduled_at(trainee)
```

Comment:
Just a suggestion but you could explose `dead_jobs` and `retry_jobs` to the view and build up the helper logic in a helper directly. Otherwise these methods look like custom controller actions here